material-ui/material-ui.d.ts

```diff
@@ -1157,7 +1163,8 @@ declare namespace __MaterialUI {
         export class TableRow extends React.Component<TableRowProps, {}> {
         }
 
-        interface TableRowColumnProps extends React.Props<TableRowColumn> {
+        interface TableRowColumnProps extends HasStyleAndClassNameProps<TableRowColumn> {
```

Comment:
Based on the `render()` method in the material-ui source code, since `td` is passed all `other`  properties, use `React.HTMLAttributesBase<TableRowColumn>` instead of `HasStyleAndClassNameProps`.